### PR TITLE
Document use of ibm burstbuffer ansible playbooks for deployment

### PR DESCRIPTION
**Pull request by meahoibm**
Document ansible playbooks for deploying and removing burstbuffer on a cluster.

### CODE REVIEW

**Comment by meahoibm** [comment]
@tgooding Please review.  For some reason you were not int the list of reviewers.

**Comment by meahoibm** [comment]
Could tgooding be added as an approver?

File: docs/source/burst-buffer/bbinstall.rst [hunk 1]
```diff
@@ -117,12 +144,14 @@ This nodelist could be generated via the xCAT commands:
 lsdef all | grep "Object name:" | cut -f 3 -d ' '
 
 
-The second file (esslist) contains a list of IP addresses and ports for each bbServer.  In the planned configuration, this would be the ESS I/O node VM IP address plus a well-known port (e.g., 9001).  To express ESS redundancy, place the two ESS’s I/O nodes within the same ESS should be placed on the same line.  E.g.,:
```
Comment (tgooding) [inline docs/source/burst-buffer/bbinstall.rst:120]: We didn't drop support for the older esslist format.  Just added an alternative in bbactivate.

Comment (meahoibm) [reply docs/source/burst-buffer/bbinstall.rst:120]: Missed this comment.  Corrected bbinstall.rst to have both previous format and newer.

**Pull request merged**